### DISCUSSION TITLE
(Android) Audio Bug

### DISCUSSION THREAD

**Discussion by MineTurtlee** [ppy/osu#23488] [Q&A]
### Type

Game behaviour

### Bug description

After playing for **a while**, somehow the audio stops playing and beatmaps wouldn't load. The timeline stops at -*:**.
Solution for me: Set audio speaker to Default(Alarm) and reset back. Or, restart game.

### Screenshots or videos

not here now

### Version

almost every versions

### Logs

no logs here, data is hidden for now...

**Comment by bdach** [comment]
Please add actual details to the issue, including

- device make and model
- video of issue presentation if possible
- the logs that are missing

**Comment by MineTurtlee** [comment]
Model: Samsung SM-A115F
Video: impossible, no osu!
Logs: nothing, "Due to security reasons, you cannot view this folder in this phone."

**Reply by ItsShamed** [reply]
Use your computer to access the files

https://github.com/ppy/osu/assets/39100084/50fa0139-d731-43a4-a6c0-a19bdd6d6212
